packages/conf-libX11/conf-libX11.1/files/pkg-osx.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ -n "$PKG_CONFIG_PATH" ] ; then
+  PKG_CONFIG_PATH=$PKG_CONFIG_PATH:
+fi
+
+PKG_CONFIG_PATH=$PKG_CONFIG_PATH/usr/local/lib/pkgconfig:/opt/X11/lib/pkgconfig pkg-config x11
```

Comment:
Will this work on windows, or it will suffer from the same issue seen here: https://github.com/ocaml/opam-repository/issues/13636 ?

Reply:
I guess it will, given that depends on conf-pkg-config

Reply:
This script only runs on macOS!

Reply:
Oops, indeed :)